python/lsst/obs/base/yamlCamera.py

```diff
@@ -40,25 +40,26 @@ def __init__(self, cameraYamlFile):
 
         plateScale = afwGeom.Angle(cameraParams["plateScale"], afwGeom.arcseconds)
         radialCoeffs = np.array(cameraParams["radialCoeffs"])/plateScale.asRadians()
-        focalPlaneToPupil = afwGeom.makeRadialTransform(radialCoeffs)
-        pupilToFocalPlane = focalPlaneToPupil.getInverse()
-        cameraTransformMap = cameraGeom.CameraTransformMap(cameraGeom.FOCAL_PLANE,
-                                                           {cameraGeom.PUPIL: pupilToFocalPlane})
-        detectorList = self._makeDetectorList(cameraParams["CCDs"], pupilToFocalPlane, plateScale)
+        focalPlaneToFieldAngle = afwGeom.makeRadialTransform(radialCoeffs)
+        fieldAngleToFocalPlane = focalPlaneToFieldAngle.getInverse()
+        cameraTransformMap = cameraGeom.TransformMap(cameraGeom.FOCAL_PLANE,
+                                                    {cameraGeom.FIELD_ANGLE: fieldAngleToFocalPlane})
```

Comment:
Each transform in the dict should transform from the reference coordinate system (FOCAL_PLANE) to the dict key (FIELD_ANGLE in this case. The old code was incorrect and you can trivially make the new code correct by eliminating the line that makes constructs `fieldAngleToFocalPlane` and using `focalPlaneToFieldAngle` here.

Reply:
No problem, not your fault at all; as you say, this code does indeed have no unit tests. DM-11924 exists to address that (and the introduction of `ci_ctio0m9` will also help).